content/en/tracing/trace_explorer/trace_view.md

```diff
@@ -84,11 +84,18 @@ To expand or collapse span descendants, click the chevron (>) icon on a row. To
 {{% /tab %}}
 {{% tab "Map" %}}
 
-{{< img src="tracing/trace_view/map.png" alt="Map" style="width:90%;">}}
+{{< img src="tracing/trace_view/trace-map.png" alt="Trace map" style="width:100%;">}}
 
-Displays a representation of all the services involved in the trace. This visualization is useful for getting a high-level overview of the services' dependencies and transaction lifecycle at a service level.
+Trace map displays a representation of all services involved in a single trace, providing an overview of the transaction lifecycle at the service level and clarifying service dependencies.
+
+Each node on the map represents a service in the transaction lifecycle. To prevent cyclic dependencies on the map, services that call back to a service already invoked by them are represented by duplicated nodes.
+
+Service nodes explicitly show the percentage of the **total execution time** which provides an overview of the trace duration breakdown at the service level.
```

Comment:
What is the percentage of? What is the "total execution time" being compared to?

```suggestion
Service nodes explicitly show the percentage of the **total execution time**, which shows the trace duration breakdown at the service level.
```